src/main/java/com/videoclub/suriken/security/model/AppUser.java

```diff
@@ -0,0 +1,62 @@
+package com.videoclub.suriken.security.model;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+public class AppUser {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String userName;
+
+    private String password;
+
+    @ManyToMany(cascade = {CascadeType.PERSIST, CascadeType.MERGE})
+    @JoinTable(name = "user_roles", joinColumns = {@JoinColumn(name = "user_id")}, inverseJoinColumns = {@JoinColumn(name = "role_id")})
```

Comment:
Nikad nisam koristio inverseJoinColumns. Zasto to ovde?

Reply:
Bukvalno samo za nazive kolona u toj dodatnoj tabeli koja prati relacije